codex-ui/src/vue/components/select/Select.vue

```diff
@@ -45,8 +46,11 @@ const defaultValue: SelectItem = props.defaultItem;
 const activeItem = ref(defaultValue);
 
 /* Main function to update selected item */
+/* Also creates new event, which could be caught outside */
+
 const updateActiveItem = (item: DefaultItem) => {
   activeItem.value = item;
+  emit('valueUpdate', activeItem.value);
```

Comment:
lets use `v-model` instead. To make Select work like native input
https://vuejs.org/guide/components/v-model

```vue

<Select
  :items="roleItems"
  :default-item="defaultRole"
  v-model="currentRole"
/>
```